src/api/features/features.api.js

```diff
@@ -423,6 +470,9 @@ async function identifyOnExternalWmsLayer(config) {
                 // nothing to do other than extracting the data
                 features = getFeatureInfoResponse.data.features
                 break
+            case APPLICATION_OGC_WMS_XML_TYPE:
+                features = parseOGCWMSFeatureInfoResponse(getFeatureInfoResponse.data)?.features
+                break
             case PLAIN_TEXT_TYPE:
                 // TODO : implement plain text parsing
                 log.error('Plain text parsing not yet implemented')
```

Comment:
if it's not too much work, I would add a small `addWarning` dispatch here, to let the user know that if he imports a plain text document, we won't parse it.

Reply:
It seems not straight forward to add. I can't import the store in the `api` directory. I will skip it for now, and treat like other '_error_' in this file with `log.error` only.